### PR TITLE
Feature/issue 4 develop snmp library

### PR DESCRIPTION
**Pull request by kawtarBK9**
* Developed SNMP client
* Updated license headers
* Updated the README.md file
* Updated the index.md in site/markdown

### CODE REVIEW

File: .idea/.gitignore
Comment (CherfaElyes) [inline .idea/.gitignore:1]: You should delete all IDEA-related directories and files: ".idea",

Comment (kawtarBK9) [reply .idea/.gitignore:1]: It's done !

**Review by NassimBtk — CHANGES_REQUESTED** [review]
Thank you @kawtarBK9, couple of small changes then we should be good.

File: src/main/java/org/sentrysoftware/snmp/client/SnmpResult.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * ╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱
+ */
+
+package org.sentrysoftware.snmp.client;
+
+public class SnmpResult
```
Comment (NassimBtk) [inline src/main/java/org/sentrysoftware/snmp/client/SnmpResult.java:25]: Please format this class.

File: src/main/java/org/sentrysoftware/snmp/client/SnmpClient.java [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+	private int snmpVersion;
+	private String authUsername;
+	private String authType;
+	private String authPassword;
+	private String privacyType;
+	private String privacyPassword;
+	private int[] retryIntervals;
+	private String contextName;
+	private byte[] contextEngineID;
+	static public final String socketType = "Standard";
```
Comment (NassimBtk) [inline src/main/java/org/sentrysoftware/snmp/client/SnmpClient.java:64]: The constant should be declared as the following:
```java
	public static final String SOCKET_TYPE = "Standard";
```

**Review by NassimBtk — CHANGES_REQUESTED** [review]

File: .idea/.gitignore
[the remark this answers is not in the document]
Comment (NassimBtk) [reply .idea/.gitignore:1]: Already done! Thank you @CherfaElyes and @kawtarBK9!

File: src/site/markdown/index.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
-# Documentation
+# SNMP Java Client
 
-This is the documentation.
+The SNMP Java Client enables you to execute basic SNMP operations, including:
+
+* Initializing the SNMP Client
+* Establishing a secure connection to a remote server
```
Comment (bertysentry) [inline src/site/markdown/index.md:6]: These come from the SSH client 😅

File: .idea/.gitignore
Comment (NassimBtk) [inline .idea/.gitignore:1]: Why is `.idea` being re-added once again, please delete it. Update your .gitignore as follows:

```
### IntelliJ IDEA ###
.idea
*.iws
*.iml
*.ipr
```

**Review by NassimBtk — CHANGES_REQUESTED** [review]

**Review by NassimBtk — APPROVED** [review]

**Pull request merged**